tools.js

```diff
@@ -29,6 +29,12 @@ module.exports = {
      return `To install yarn, see https://github.com/yarnpkg/yarn/releases/tag/v${v}`;
     }
   },
+  pnpm: {
+    getVersion: "pnpm --version",
+    getInstallInstructions(v) {
+     return `To install npm, run \`npm install -g pnpm@${v}\``;
```

Comment:
```suggestion
     return `To install pnpm, run \`npm install -g pnpm@${v}\``;
```

Reply:
Hehe my bad

Reply:
Ah, missed this. I will push a fix as v4.2.1. Thank you.